backends/postgres/postgres_backend.go

```diff
@@ -761,13 +761,13 @@ func (p *PgBackend) listen(ctx context.Context, queue string) (c chan string, re
 	go func(ctx context.Context) {
 		conn, err := p.pool.Acquire(ctx)
 		if err != nil {
-			p.logger.Error("unable to acquire new listener connnection", "error", err)
+			p.logger.Error("unable to acquire new listener connection", "error", err)
 			return
 		}
 		defer p.release(ctx, conn, queue)
 
 		// set this connection's idle in transaction timeout to infinite so it is not intermittently disconnected
-		_, err = conn.Exec(ctx, fmt.Sprintf("SET idle_in_transaction_session_timeout = '0'; LISTEN %s", queue))
+		_, err = conn.Exec(ctx, fmt.Sprintf(`SET idle_in_transaction_session_timeout = '0'; LISTEN %q`, queue))
```

Comment:
We'll need quotes around the queue name here too.

Reply:
`%q` will quote the string when its formatted, so this is already set!

Reply:
Giddyup!